SystemServices/SystemServices.cpp

```diff
@@ -1790,11 +1790,14 @@ namespace WPEFramework {
             std::vector<string> lines;
 
 	    if (!Utils::fileExists(FWDNLDSTATUS_FILE_NAME)) {
-		    populateResponseWithError(SysSrv_FileNotPresent, response);
-		    returnResponse(retStat);
+                //If firmware download file doesn't exist we can still return the current version
+                response["currentFWVersion"] = getStbVersionString();
```

Comment:
Pls check if getStbVersionString() returns "unknown" in which case return empty ("") and then return success as false.